discovery_engine_core/core/src/stack/ops.rs

```diff
@@ -50,15 +50,15 @@ pub trait Ops {
     /// tailored to the user's interests.
     async fn new_items(&self, key_phrases: &[KeyPhrase]) -> Result<Vec<Article>, GenericError>;
 
-    /// Filter `articles` based on `current` documents.
+    /// Filter `articles` based on `stack` documents.
     fn filter_articles(
         &self,
-        current: &[Document],
+        stack: &[Document],
         articles: Vec<Article>,
     ) -> Result<Vec<Article>, GenericError>;
 
     /// Merge current and new items.
```

Comment:
to align with the other renaming
```suggestion
    /// Merge stacked and new items.
```